From 42310a7851049d7706c66874f363ce7ef73655df Mon Sep 17 00:00:00 2001
From: Jacopo De Simoi <wilderkde@gmail.com>
Date: Thu, 1 May 2014 16:46:46 -0400
Subject: [PATCH 09/16] Do not popup during initialization

The notifier should not popup if devices are detected during the
applet initialization phase (e.g. on plasma startup)
CCBUG: 318061
---
 .../applets/devicenotifier/package/contents/ui/devicenotifier.qml       | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plasma/generic/applets/devicenotifier/package/contents/ui/devicenotifier.qml b/plasma/generic/applets/devicenotifier/package/contents/ui/devicenotifier.qml
index 9b6132e..473036f 100644
--- a/plasma/generic/applets/devicenotifier/package/contents/ui/devicenotifier.qml
+++ b/plasma/generic/applets/devicenotifier/package/contents/ui/devicenotifier.qml
@@ -74,7 +74,7 @@ Item {
                     (devicesType == "removable" && data[last] && data[last]["Removable"] == true) ||
                     (devicesType == "nonRemovable" && data[last] && data[last]["Removable"] == false)) {
                     updateTooltip();
-                    if (expand) {
+                    if (expand && hpSource.data[last]["added"]) {
                         expandDevice(last)
                     }
                     last = "";
-- 
2.0.0.rc2

